app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java

```diff
@@ -318,7 +327,11 @@ private NagiosExtension convertToEbeanNagiosExtension(final models.internal.Nagi
          */
         @Override
         public PagedList<models.ebean.Alert> createAlertQuery(final AlertQuery query) {
+            if (query.getOrganization() == null) {
```

Comment:
For those not on the email, the options are refactoring to a builder pattern or putting the organization in the constructor.  Given that both of these are X changes, we're free to do either one.  Our general preference is to use builders because we can easily add validation rules with the OVal framework.  However, in this case, a completely "empty" query was acceptable (unlike with the builders).  This is the first and the only foreseeable required member.  So, I think I'm going to add it as a constructor arg and hope that we don't see any more required args.